docs/tasks/debug-application-cluster/run-command-container-kubectl-exec.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caesarxuchao
+- mikedanese
+title: Running Commands in a Container using kubectl exec
```

Comment:
There is also a Task section on kubectl.  Where do you think this should go?  Should we have it appear in the ToC in both locations perhaps?